.vscode/launch.json

```
@@ -0,0 +1,28 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Debug validation test",
+            "type": "cppdbg",
+            "request": "launch",
+            "program": "${command:cmake.buildDirectory}/bin/power_grid_model_validation_tests",
+            "args": [
+                "--test-case=Validation test single",
+                "--subcase=power_flow/1os2msr-sym-newton_raphson"
```

maybe it's better to default to running all test cases with a comment that one can select subcases? e.g.
```suggestion
                // "--test-case=Validation test single",
                // "--subcase=power_flow/1os2msr-sym-newton_raphson"
```